src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -124,17 +124,7 @@ bool OutputStateMachineEngine::ActionPrintString(const wchar_t* const rgwch, con
 bool OutputStateMachineEngine::ActionPassThroughString(const wchar_t* const rgwch,
                                                        _In_ size_t const cch)
 {
-    bool fSuccess = true;
-    if (_pTtyConnection != nullptr)
-    {
-        std::wstring wstr = std::wstring(rgwch, cch);
-        auto hr = _pTtyConnection->WriteTerminalW(wstr);
-        LOG_IF_FAILED(hr);
-        fSuccess = SUCCEEDED(hr);
-    }
-    // If there's not a TTY connection, our previous behavior was to eat the string.
-
-    return fSuccess;
+    return _dispatch->PassThroughString(rgwch, cch);

```

Comment:
this also removes the need for _pTtyConnection and ITerminalConnection, right? Since WriteTerminalW was like its only method

Reply:
IIRC `ITerminalConnection` was actually used in the passthrough prototype I had, so we might want to keep it around for now. I can try pulling out the _pTtyConnection though.